### PR TITLE
Control clean

### PR DESCRIPTION
**Pull request by MengTC**

### CODE REVIEW

File: modules/control/control_wheel_angle_real.h [hunk 1]
```diff
@@ -1,98 +0,0 @@
-﻿#pragma once
```
Comment (mickeyouyou) [inline modules/control/control_wheel_angle_real.h:1]: please remove include in `modules/control/control_component.cc`

Comment (MengTC) [reply modules/control/control_wheel_angle_real.h:1]: ok, removed

File: modules/control/control_component.cc [hunk 2]
```diff
@@ -104,167 +101,77 @@ double ControlComponent::PidSpeed() {
...
+  if (lat_dev_mgs < -3.5){
+    wheel_target = 20.0;    
+  } else if (lat_dev_mgs > 3.5) {
+    wheel_target = -20.0;
+  } else if (std::abs(lat_dev_mgs) < 0.1) {    
+    wheel_target = target_last;
+  } else if (std::abs(lat_dev_mgs) <= 3.5) {
+    wheel_target = 0;
+  }
+  target_last = wheel_target;
```
Comment (mickeyouyou) [inline modules/control/control_component.cc:115]: target_last 并没有区分前后，这个地方应该有问题

File: modules/control/control_component.cc [hunk 3]
```diff
@@ -104,167 +101,77 @@ double ControlComponent::PidSpeed() {
...
-          } else if (std::abs(rear_lat_dev_mgs) < 0.1) {
-            cmd->set_rear_wheel_target(rear_target_pre);
-          } else {
-            cmd->set_rear_wheel_target(0);
-            rear_target_pre = 0;
+        rear_wheel_target = 0;
+
+        // TODO: test wakeup with original mgs data
+        if (!front_wheel_wakeup) {
+          if (is_received) {
```
Comment (mickeyouyou) [inline modules/control/control_component.cc:156]: if (!front_wheel_wakeup && is_received)

Comment (MengTC) [reply modules/control/control_component.cc]: ok

File: modules/control/control_component.cc [hunk 4]
```diff
@@ -104,167 +101,75 @@ double ControlComponent::PidSpeed() {
   return std::isnan(torque) ? 0 : torque;
 }
 
+double ControlComponent::GetSteerTarget(float lat_dev_mgs, double & target_last) {
```
Comment (mickeyouyou) [inline modules/control/control_component.cc:104]: FLAGS_desired_v  获取不到，需要从控制配置获取

File: modules/control/control_component.cc [hunk 5]
```diff
@@ -104,167 +101,75 @@ double ControlComponent::PidSpeed() {
...
-            drivemotor_torque = PidSpeed();
-          }
-          // TODO: drivemotor_torque to config
-          if (drivemotor_torque > 46.0) {
-            drivemotor_torque = 46.0;
-          }
-          cmd->set_torque(drivemotor_torque);
+      } else {
+        if (cmd->pad_msg().action() == DrivingAction::START) {
+          drivemotor_torque = (PidSpeed() < 46.0) ? PidSpeed() : 46.0;
```
Comment (mickeyouyou) [inline modules/control/control_component.cc:139]: 第二次PidSpeed()反馈值，拿到的还是同一个力矩值吗？

File: modules/control/control_component.cc (hunk 4 above)
Comment (MengTC) [reply modules/control/control_component.cc:104]: ok

File: modules/control/control_component.cc (hunk 5 above)
Comment (MengTC) [reply modules/control/control_component.cc]: 改成了
drivemotor_torque = PidSpeed();
drivemotor_torque = (drivemotor_torque < 46.0) ? drivemotor_torque : 46.0;

**Review by mickeyouyou — APPROVED** [review]

**Pull request merged**